common/persistence/sql/sqlplugin/interfaces.go

```diff
@@ -524,6 +545,7 @@ type (
 		WriteLockShards(ctx context.Context, filter *ShardsFilter) (int, error)
 
 		InsertIntoTasks(ctx context.Context, rows []TasksRow) (sql.Result, error)
+		InsertIntoTasksWithTTL(ctx context.Context, rows []TasksRowWithTTL) (sql.Result, error)
```

Comment:
Can you add some comments why there are two similar APIs and the relation ship with SupportsTTL()? I think it will make it easier for anyone want to implement this interface can is not confused.